src/server/game/Conditions/ConditionMgr.h

```diff
@@ -127,7 +127,7 @@ enum ConditionSourceType
     CONDITION_SOURCE_TYPE_SPELL                          = 17,
     CONDITION_SOURCE_TYPE_SPELL_CLICK_EVENT              = 18,
     CONDITION_SOURCE_TYPE_QUEST_AVAILABLE                = 19,
-    // Condition SourceType 20 unused                    = 20,
+    CONDITION_SOURCE_TYPE_UNUSED_XX                      = 20,
```

Comment:
why not `CONDITION_SOURCE_TYPE_UNUSED_20` ?

Reply:
Oh that was my bad, I didn't think of that x)